src/style.css

```diff
@@ -1 +1,94 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir, Arial, sans-serif;
```

Comment:
The `font-family` property in the `@font-face` rule is not used correctly. It should be used to define a name for the font, not to specify a list of font families. The fallback fonts should be specified in the `font-family` property where you use this font. Please, correct it.